packages/components/package.json

```diff
@@ -31,6 +31,8 @@
   "dependencies": {
     "@owlui/button": "^0.0.19",
     "@owlui/card": "^0.0.19",
+    "@owlui/dropdownlist": "^0.0.0",
```

Comment:
Just a reminder to remove all references to `dropdownmenu` before putting this one for final review.